<launch>
  <include file="$(find-pkg-share bitbots_parameter_blackboard)/launch/parameter_blackboard.launch" />

  <include file="$(find-pkg-share bitbots_robot_description)/launch/load_robot_description.launch" />

  <!-- launch the base footprint -->
  <node pkg="humanoid_base_footprint" exec="base_footprint" output="screen">
    <param name="support_state_topics" value="[walk_support_state, dynamic_kick_support_state]"/>
  </node>

  <!-- launch the odometry -->
  <include file="$(find-pkg-share bitbots_odometry)/launch/odometry.launch" />

  <node name="rviz2" pkg="rviz2" exec="rviz2" output="screen"
    args="-d $(find-pkg-share wolfgang_description)/config/wolfgang.rviz" />

  <node name="motor_goals_viz_helper" pkg="bitbots_utils" exec="motor_goals_viz_helper.py" args="--kick" />

  <node pkg="bitbots_dynamic_kick" exec="KickNode" output="screen">
    <param from="$(find-pkg-share bitbots_dynamic_kick)/config/kick_config.yaml" />
  </node>
</launch>
